cli/cli/commands/feedback/feedback.go

```diff
@@ -24,7 +21,15 @@ const (
 		"See below for the many ways you can get in touch with us.\n\n" +
 		"TIP: You can quickly type and send us feedback directly from the CLI. For example, " +
 		"`kurtosis feedback \"I enjoy the enclave naming theme\"` will open the Kurtosis GitHub " +
-		"\"Choose New Issue\" page with the description pre-filled with \"I enjoy the enclave naming theme\"."
+		"\"Choose New Issue\" page with the description pre-filled with \"I enjoy the enclave naming theme\".\n\n" +
+		"This command can be used to deliver feedback to the Kurtosis team!\n\n" +
+		"* Pass in your feedback as an argument using double quotations (e.g. kurtosis feedback \"my feedback\"), and press enter; this will open our GitHub Issues templates, pre-filled with your feedback/arg.\n" +
+		"* Pass in the --email flag to open a draft email, pre-filled with your feedback/arg, to send to feedback@kurtosistech.com.\n" +
+		"* Pass in the --calendly flag to open our Calendly link to schedule a 1:1 session with us for feedback and questions you may have!\n\n" +
```

Comment:
All of this sounds good! we are duplicating some information given by the flags themselves I suppose. Is that okay?

Reply:
yes, because there are small differences between both places